src/main/java/com/autotune/common/bulk/BulkServiceValidation.java

```diff
@@ -0,0 +1,95 @@
+package com.autotune.common.bulk;
+
+import com.autotune.analyzer.serviceObjects.BulkInput;
+import com.autotune.analyzer.serviceObjects.BulkJobStatus;
+import com.autotune.common.data.ValidationOutputData;
+import com.autotune.common.datasource.DataSourceInfo;
+import com.autotune.common.datasource.DataSourceOperatorImpl;
+import com.autotune.common.utils.CommonUtils;
+import com.autotune.database.dao.ExperimentDAOImpl;
+import com.autotune.database.service.ExperimentDBService;
+import com.autotune.database.table.lm.KruizeBulkJobEntry;
+import com.autotune.utils.KruizeConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeParseException;
+
+public class BulkServiceValidation {
```

Comment:
Adding a Javadoc could be a bit helpful for the devs to understand about this class.